Providers/Scripts/2.4x-2.5x/Scripts/nxOMSPlugin.py

```diff
@@ -163,6 +172,19 @@ def Set(WorkspaceID, Plugins):
             # log error - neither conf nor plugin directory was found in IP to set
             LG().Log('ERROR', plugin['PluginName'] + ' contains neither plugin nor conf')
             return [-1]
+
+        # Some Plugins (e.g. Security Baseline) have arch specific files
+        if os.path.isdir(plugin_arch_dir):
+            if plugin['Ensure'] == 'Present':
+                # copy all files under plugin
+                copy_all_files(plugin_arch_dir, PLUGIN_PATH, True)
+            elif plugin['Ensure'] == 'Absent':
+                # NO-OP as we do *NOT* remove common plugins
+                pass
+            else:
+                # log error Ensure value not expected
+                LG().Log('ERROR', 'Ensure value: ' + plugin['Ensure'] + ' not expected')
+                return [-1]
```

Comment:
If the `plugin_arch_dir` path does not exist, I would suggest logging at least an INFO message